clients/banking/src/components/AccountArea.tsx

```diff
@@ -196,6 +196,10 @@ export const AccountArea = ({ accountMembershipId }: Props) => {
     typeof value === "boolean" ? Option.Some(value) : Option.None(),
   ).some(isValid => isValid);
 
+  const hasPvidOrQesValidation =
```

Comment:
this name isn't exactly right because there is also french account country condition. And the name could become even longer if we add more complex conditions.  
Maybe we could name this just `firstTransactionNotRequired` which is similar to my suggestion for `AccountActivationPage`.  
What do you think?